### PR TITLE
Adding new default_host parameter

### PR DESCRIPTION
**Pull request by sharumpe**
I know you're not really set up for submissions at this point, but just in case this is helpful, here's a small change I've made to a fork that I'd love to see in the base.  :) 

The 'norm' at our location is to have syslog use the hostname instead of the fqdn.  Setting the sourcetype to syslog will result in those logs having the "correct" hostname, while others may not be parsed and will end up with the fqdn, which makes for more troublesome searches for that host.

TL;DR: setting the default_host rather than using the parsed value means that confusion can be resolved.

I have tested this on CentOS 7.2 and Ubuntu 14.04.

Thanks!

### CODE REVIEW

**Comment by sharumpe** [comment]
OK, I have just made another commit (to add support for the Suse osfamily) which is a very small commit but I didn't realize it would be rolled up with this.

**Comment by bastelfreak** [comment]
Hi @sharumpe, are you still interested in this PR? If so, please rebase against our latest master.

**Comment by dhoppe** [comment]
@sharumpe Ping

**Comment by bastelfreak** [comment]
Hello. I'm going to close this PR due to inactivity. Please reopen and rebase it if you're still interested in it.
